ui/src/components/App.js

```diff
@@ -10,7 +9,7 @@ class App extends React.Component {
       <Router>
         <div className='App'>
           <Switch>
-            <Route exact path="/" component={LandingPage} />
+            <Route exact path="/" component={AppDetailedView} />
             <Route path="/app/:id" component={AppDetailedView} />
```

Comment:
I think the path for the app detailed view should be `apps/:id`, now `/app/id`. This follows the same pattern as the API and I think it is the standard convention anyway.